### ISSUE TITLE
Why use axis 0 as default for deconv in pytorch_quantization?

### ISSUE THREAD

**Issue by hangxu124** [NVIDIA/TensorRT#2380]
## Description

I realize that in the last commit (https://github.com/NVIDIA/TensorRT/commit/e34b33ef748d8a24fa6a0716225d09b23c9880c3) you change the default QuantDescriptor in ConvTranspose per channel quantization, from axis=1 to axis=0. As I know, the per channel quantization should be used per output_channel, but in ConvTranspose, axis0 is input_channel and axis1 is output_channel. I'm wondering if there are some special reasons for you to change it from axis1 to axis0.

## Environment

**TensorRT Version**:  
**NVIDIA GPU**: 
**NVIDIA Driver Version**: 
**CUDA Version**: 
**CUDNN Version**: 
**Operating System**: 
**Python Version (if applicable)**: 
**Tensorflow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 
**Baremetal or Container (if so, version)**: 

## Relevant Files

https://github.com/NVIDIA/TensorRT/commit/e34b33ef748d8a24fa6a0716225d09b23c9880c3

**Comment by zerollzeng** [comment]
@ttyio ^ ^

**Comment by nvpohanh** [comment]
ConvTranspose is just the opposite (back prop) of Conv, so since we use per-output-channel quant for Convs, the kernels assume per-input-channel quant for ConvTranspose.

**Comment by nvpohanh** [comment]
We don't support per-output-channel quant for ConvTranspose

**Comment by ttyio** [comment]
Closing since no activity for more than 3 weeks, please reopen if you still have question, thanks!

**Comment by nvpohanh** [comment]
Sorry, after close investigation, we found that axis=1 was the correct approach for per-out-channel quantization for ConvTranspose operations. We will update the quantization tool in next release to fix this. For now, please revert the changes in https://github.com/NVIDIA/TensorRT/commit/e34b33ef748d8a24fa6a0716225d09b23c9880c3
